waspc/cli/Wasp/Cli/Command/Common.hs

```diff
@@ -3,6 +3,10 @@ module Wasp.Cli.Command.Common
     findWaspProjectRoot,
     waspSaysC,
     alphaWarningMessage,
+    toWaspMessage,
+    toWaspStartMessage,
+    toWaspSuccessMessage,
+    toWaspFailureMessage,
   )
```

Comment:
The only other thing I was considering was instead of having these be String formatting helpers, they could be distinct `waspSays` functions. But that would preclude their usage in other contexts, like throwing errors and such. It also feels better to have a single `waspSays` and just modify the arguments (to me at least).

Reply:
Good point on location, I will move them into something not-`Command` related as you suggested. 👍🏻

Reply:
I also went back and forth between `to` and `as`, since it was a suggested alternative I will update that too.

Reply:
Updated here: https://github.com/wasp-lang/wasp/commit/526f6e93b45a88ddf99d59ac3bb92d1f244a8d59